ember-highcharts/src/components/high-charts.ts

```diff
@@ -243,6 +243,19 @@ export default class HighCharts extends Component<HighChartsSignature> {
       SolidGauge.default(Highcharts);
     }
 
+    if (
+      this.args.chartOptions?.chart?.type === 'treegraph' ||
```

Comment:
So treegraph and treemap both require `Treemap`?

Reply:
Yes.


Treemap
https://www.highcharts.com/docs/chart-and-series-types/treemap
> The treemap chart requires the filemodules/treemap.js.

Treegraph
https://www.highcharts.com/docs/chart-and-series-types/treegraph-chart
> In order to use this series type, you need to load the modules/treemap.js and modules/treegraph.js modules.